packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -1011,6 +1011,25 @@ new lambda.Function(this, 'Lambda', {
 });
 ```
 
+Providing a custom, user controlled log group this way is not yet supported in all regions, namely GovCloud and CN.
```

Comment:
This is needlessly and negatively specific; also it will go out of date once the feature rolls out globally.

I would phrase it positively instead:

> Providing a user-controlled log group was rolled out to commercial regions on 2023-XX-YY. If you are deploying to another type of region, please check regional availability first.

Reply:
Or "as of 2024-01-31 the feature is available in all commercial regions. If you are ..."